validator/client/src/main/java/tech/pegasys/teku/validator/client/loader/ValidatorLoader.java

```diff
@@ -72,11 +72,15 @@ private ValidatorLoader(
 
   // synchronized to ensure that only one load is active at a time
   public synchronized void loadValidators() {
-    final Map<BLSPublicKey, ValidatorProvider> validatorProviders = new HashMap<>();
-    validatorSources.forEach(source -> addValidatorsFromSource(validatorProviders, source));
-    MultithreadedValidatorLoader.loadValidators(
-        ownedValidators, validatorProviders, graffitiProvider);
-    slashingProtectionLogger.protectionSummary(ownedValidators.getValidators());
+    try {
+      final Map<BLSPublicKey, ValidatorProvider> validatorProviders = new HashMap<>();
+      validatorSources.forEach(source -> addValidatorsFromSource(validatorProviders, source));
+      MultithreadedValidatorLoader.loadValidators(
+          ownedValidators, validatorProviders, graffitiProvider);
+      slashingProtectionLogger.protectionSummary(ownedValidators.getValidators());
+    } catch (final Exception ex) {
+      throw new ValidatorLoaderException(ex);
+    }
```

Comment:
this wraps with our new exception, but it doesn't really add value... we were already checking the main cases... so the only difference is now we always get a ValidatorLoaderException in this specific case, but for all other stages of loading, we don't really get more insight. I'm not sure it's that valuable unless we're trying to split out config exceptions here from other config exceptions.

Reply:
I want to clearly separate keys loading and other exceptions. If you log with "keystore" wording when `scheduleValidatorsDuties` or `initializeValidatorRestApi` fails, it could be confusing

Reply:
Though `initializeValidatorRestApi` is kind of a part of a keystore too...
I didn't like full exception name with package name when it's not `InvalidConfigurationException`

Reply:
actually very academic. the known ones were keystore, and given the error string and stack trace at exit of unkown we'd have adequate information...